src/Okta.Idx.Sdk.E2ETests/Configuration.cs

```diff
@@ -1,14 +1,14 @@
 ﻿namespace Okta.Idx.Sdk.E2ETests
 {
-    public class TestConfig : ITestConfig
+    public class Configuration : IConfiguration
     {
         public string UserPassword { get; set; }
         public string A18nApiKey { get; set; }
         public string A18nProfileId { get; set; }
         public string A18nProfileTag { get; set; }
         public int IisPort { get; set; }
-        public string SiteUrl { get; set; }
-        public string MfaRequiredGroupName { get; set; }
-        public TestUserProperties TestUser { get; set; }
+        public string SiteUrl { get; set; }
+        public string MfaRequiredGroup { get; set; }
+        public string PhoneEnrollmentRequiredGroup { get; set; }
```

Comment:
At some point I'd like to see documentation comments for these properties (not necessarily as part of this PR).